src/options/arith_options.toml

```diff
@@ -409,6 +409,14 @@ name   = "Arithmetic theory"
   default    = "true"
   help       = "incremental linearization approach to non-linear"
 
+[[option]]
```

Comment:
Can we make nlExt a three valued mode?  Would avoid having a new flag and would ensure the mode was clearer.

Reply:
For this change, I unfortunately needed to touch a bunch of regressions...